force-app/main/default/lwc/libsChartjs/libsChartjs.html

```diff
@@ -1,8 +1,6 @@
 <template>
     <lightning-card title="LibsChartjs" icon-name="custom:custom19">
-        <div class="slds-m-around_medium">
-            <canvas class="donut" lwc:dom="manual"></canvas>
-        </div>
+        <div class="chart slds-m-around_medium" lwc:dom="manual"></div>
```

Comment:
Chart.js modifies the parent of the target `<canvas>`. This triggers a warning because the parent `<div>` is not decorated with `lwc:dom=manual`.